packages/hurl/src/runner/response.rs

```diff
@@ -78,8 +78,8 @@ pub fn eval_asserts(
                 });
             }
             Ok(expected) => {
-                let header_name = &header.key.value;
-                let actuals = http_response.get_header_values(header_name);
+                let header_name = eval_template(&header.key, variables).unwrap(); //FIXME FAB
```

Comment:
Maybe put a longer comment?  `// FIXME FAB` doesn't give a lot of context....

Reply:
it was for this PR in fact, I'm going to fix it. 
It was about the `unwrap`